flutter/lib/ui/geometry.dart

```diff
@@ -769,48 +886,62 @@ class RRect {
   Rect get outerRect => new Rect.fromLTRB(left, top, right, bottom);
 
   /// The non-rounded rectangle that fits inside this rounded rectangle by
-  /// touching the middle of each curved corner.
+  /// touching the middle of the inner-most curved corners.
```

Comment:
this isn't quite an accurate description. Consider an RRect that's got zero radii except one corner. The safeInnerRect will touch the outer corner on the side opposite the non-zero radius corner.